stock_move_location/wizard/stock_move_location_line.py

```diff
@@ -0,0 +1,136 @@
+# Copyright (C) 2011 Julius Network Solutions SARL <contact@julius.fr>
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+from odoo.tools import float_compare
+
+
+class StockMoveLocationWizardLine(models.TransientModel):
+    _name = "wiz.stock.move.location.line"
+    _description = "Wizard move location line"
+
```

Comment:
```suggestion
   _rec_name = "product_id
```

Reply:
This will help for choosing line when user chooses the product they want. Currently showing with ID.
![Screenshot from 2022-12-01 12-29-04](https://user-images.githubusercontent.com/45355704/204978844-b808bcbd-41d9-48e2-b28c-0658046f74d5.png)